frontend/src/context/SpotlightifyContext.tsx

```
@@ -31,73 +31,97 @@ function spotlightifyReducer(
-      const pushedCommandHistory = [...state.commandHistory, action.payload];
+      const pushedCommandHistory = [
+        ...state.commandStack,
+        {
+          command: action.payload.command,
+          options: action.payload.options || {},
+        },
+      ];
+      if (pushedCommandHistory.length > 1) {
+        const lastCommand =
+          pushedCommandHistory[pushedCommandHistory.length - 2];
+        if (lastCommand && lastCommand.options) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)